src/actions/breakpoints/syncBreakpoint.js

```diff
@@ -57,8 +57,10 @@ function createSyncData(
   text: string,
   originalText: string
 ): BreakpointSyncData {
+  const { options } = pendingBreakpoint;
   const overrides = {
     ...pendingBreakpoint,
+    ...options,
```

Comment:
ohh ... that 's what you mean.

This actually doesn't do that , this just spreads `options` out into the `overrides` object (which is just a flat object of properties to be overridden)
`overrides` gets passed into `createBreakpoint` on line 69 here https://github.com/devtools-html/debugger.html/pull/7818/files#diff-e69cbcbeb2691ef3d0839e5403221a3eR69

The `createBreakpoint` function already uses the `override` properties here
https://github.com/devtools-html/debugger.html/blob/master/src/utils/breakpoint/index.js#L124-L159

The main issue here was that doing `...pendingBreakpoint` does not deep spread , so to override the `options` properties we  need to spread it out as well.

Reply:
ahh - i should read more closely

Reply:
would this work as well?
https://gist.github.com/2a30879d003c5f8af5f6833cf9617549

Reply:
Yeah .. i tried that, Causes an error on reload because here https://github.com/devtools-html/debugger.html/blob/b0ceadd181757fc1a88161aff0261c692e5b0968/src/actions/breakpoints/addBreakpoint.js#L168 just the options are passed in.

i also think the `overrides` should be kept flat, since its just properties to override.  (just my preference)

Reply:
I'd rather the overrides stay in the shape of a breakpoint. That way it is easier to flow type.

Perhaps we should do: `createBreakpoint(location, { options });`

Reply:
👍🏾🙂